src/styles/header/header.scss

```diff
@@ -0,0 +1,62 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: $white-color;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.nav {
```

Comment:
Each BEM block should be in the separate file

Reply:
still not fixed
<img width="769" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/70220667/456e3d27-bdb3-4c7f-8c1a-b137c7c45a38">